src/ngraph/frontend/onnx_import/utils/convpool.cpp

```diff
@@ -37,28 +37,40 @@ namespace ngraph
 
             namespace detail
             {
-                Strides get_strides_helper(const Node& node,
-                                           const std::string& name,
-                                           const Shape& kernel_shape)
+                /// \brief              Helper method used to read vector attribute
+                /// \note               Default value is vector of size spatial dims filled with
+                ///                     ones
+                ///
+                /// \param   node       Node from which attribute is read
+                /// \param   attr_name  Attribute name (such as `strides`, `dilations`)
+                ///
+                /// \return             Read vector attribute if available or default value
+                Strides get_attribute_value(const Node& node, const std::string& attr_name)
```

Comment:
This API is confusing currently:
`Strides dilations = get_attribute_value(node, 'dilations');`

Why am I creating a `Strides` object when I ask for `'dilations'`?
It would be easier to understand if this function returned the `std::vector<std::size_t>` type...

Reply:
Done, `std::vector<std::size_t>` used